src/compiler/emitter.ts

```diff
@@ -862,7 +862,37 @@ namespace ts {
                     case SyntaxKind.EnumMember:
                         return emitEnumMember(<EnumMember>node);
 
-                    // JSDoc nodes (ignored)
+                    // JSDoc nodes

```

Comment:
I guess so, but this may change if we enable jsdoc types (or other info) in our normal emit. (I believe this is a long-standing request from closure and related tools)